package.json

```diff
@@ -64,17 +64,17 @@
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-react": "^7.15.0",
     "eslint-plugin-standard": "^4.0.1",
-    "i18next": "^17.0.17",
-    "i18next-browser-languagedetector": "^3.0.3",
-    "i18next-icu": "^1.1.2",
-    "i18next-xhr-backend": "^3.2.0",
+    "i18next": "^19.1.0",
+    "i18next-browser-languagedetector": "^4.0.1",
+    "i18next-icu": "^1.3.1",
+    "i18next-http-backend": "^1.0.15",
```

Comment:
`i18next-xhr-backend` is deprecated, switched to `i18next-http-backend`, will do the same in ipfs-webui and explore.ipld.io